rust/parse_ast/src/lib.rs

```diff
@@ -29,6 +31,7 @@ impl Emitter for TestEmitter {
   }
 }
 
+#[wasm_bindgen]
```

Comment:
I'm not sure if it affects the napi build, should we create a new package for the wasm build?

Reply:
Yes, please move it into a separate Cargo workspace, just how I did it for the Napi bindings, even if it means creating a wrapper function that just forwards results. At the very least, this allows to keep dependencies separate to speed up builds, and it probably would cause problems for the Napi build anyway.

Reply:
done